instructions/level1.md

```diff
@@ -244,6 +244,14 @@ To validate our form we will need to follow next steps:
   TODO: 2. Get value from field description and validate that it is not 
   empty, has more then 2 characters, less then 250 characters and consist 
   only from letters and numbers.
+
+  If you remember from before, you used the syntax
+
+  `var onlyLetters = /^[A-z]+$/.test(value);`
+
+  to indicate that you only want letters. In order to define only numbers,
+  you can add `0-9` inside the brackes, like `[A-z0-9]`. This allows any
+  letter, and any number. You might want to allow spaces as well!
```

Comment:
This was another common one, we just gave them the regex without really explaining how to change it. I've tried to address that here a bit, give them the update and suggest more changes they could make.